### ISSUE TITLE
Wordlist after log out does not clear entries

### ISSUE THREAD

**Issue by monzug** [alpheios-project/alpheios-core#449]
Tested in Chrome on PC in build Alpheios Reading Tools 3.3.0 build qa.20200611246
Alpheios Components 3.3.0-qa.20200611238

log out from user monzug. the wordlist icon is still highlighted and there are some words saved in memory from previous sessions and previous days (before installing latest build and clearing my cache). by reloading the page, the icon goes to inactive state.
see screenshot
![wordlist-after-logout](https://user-images.githubusercontent.com/41396793/84389625-536fd880-abc4-11ea-9fd1-50fe282f643f.JPG)

**Comment by balmas** [comment]
@kirlat maybe we lost a call to init the list upon logout? Or an error is interfering with it being executed? It should be handed by this watch on the `state.auth.isAuthenticated` state variable:

```
    this.authUnwatch = this.store.watch((state) => state.auth.isAuthenticated, (newValue, oldValue) => {
      this.initUserDataManager(newValue)
    })
```

**Comment by kirlat** [comment]
@monzug, is this happening with webextension on The Latin Library site? When you press the "Log out" button, do you see an Auth tab changing its state accordingly (i.e. the user name disappears and the button changes to the "Log in")? Do you see any error messages in the browser's console after the "Log out" button is pressed?

I do observe an error while trying to log out in the embed lib with the test Auth0 config. I'm not sure if it's the same or different.

**Comment by monzug** [comment]
yes  I see state changing from log in to log out. I haven't checked the console log. let me see if I can see any error.

**Comment by kirlat** [comment]
@monzug, one more question: are you using a test configuration to log in (i.e. it logs you in without asking for the username and password) or do you use a production configuration with "real" accounts?

**Comment by monzug** [comment]
I can't reproduce the log out with wordlist anymore. I guess it's one of those oddities with wordlist. @kirlat, do you want to keep this issue to fix the error that you described above?

**Comment by monzug** [comment]
sign in with google with saved psw

**Comment by balmas** [comment]
> I can't reproduce the log out with wordlist anymore. I guess it's one of those oddities with wordlist. @kirlat, do you want to keep this issue to fix the error that you described above?

I don't like that kind of problem :-(

**Comment by kirlat** [comment]
I tried to logged in with my Google account but I was blocked, even after entering a password, a secondary email and a verification code send by SMS. Google, I understand you less and less now!
I was able to log int with Apple ID and I did not experience any issues with the logout.

I've submitted a fix for an issue with logging out as a test user that I discovered during testing.

**Comment by kirlat** [comment]
A fix for the issue I've found has been merged to master.

**Comment by balmas** [comment]
Alpheios Reading Tools 3.3.0 build qa.20200612198
Alpheios Components 3.3.0-qa.20200611509

**Comment by monzug** [comment]
I haven't been able to reproduce the scenario above. when logged out, wordlist is not longer populated and the icon is not highlighted. Tested in Chrome and FF/PC.

**Comment by monzug** [comment]
I experienced a similar scenario today in Chrome on mac. logged out from the session and the wordlist does not clear.  see screenshot

**Comment by balmas** [comment]
I was able to reproduce this in Chrome on Mac. Investigating.

**Comment by balmas** [comment]
Hmm. I could only reproduce it once. Not so far again.

**Comment by monzug** [comment]
me too, two more times. the interesting thing is that the wordlist is not
related to previous user wordlist or maybe only a part of it?

**Comment by monzug** [comment]
could it be that I have a huge wordlist that takes some times to load and
that when I am done with the testing and I log out, the wordlist actually
is still loading some section? that would explain the only syriac words in
wordlist...

**Comment by balmas** [comment]
that is, in theory, possible.

**Comment by balmas** [comment]
I think in any event this is probably not new with the release, so I'd like to postpone fixing it until the next one. If the above scenario is what is causing it, fixing #18 would be required to resolve it.

**Comment by monzug** [comment]
totally agree

**Comment by monzug** [comment]
today, I logged in to Safari and found a similar scenario. I turned Alpheios on and off and still the wordlist was populated and session was expired. see screenshots
![Screen Shot 2020-06-18 at 4 26 54 PM](https://user-images.githubusercontent.com/41396793/85036420-6d8d5600-b184-11ea-829b-d5209ae77e49.png)
![Screen Shot 2020-06-18 at 4 27 15 PM](https://user-images.githubusercontent.com/41396793/85036433-70884680-b184-11ea-91b6-1eea5d762928.png)

**Comment by monzug** [comment]
FF 77 on Mac. logged out from user test, wordlist is still populated
 Alpheios Reading Tools 3.3.0 build qa.20200618484
Alpheios Components 3.3.0-qa.20200618482

**Comment by kirlat** [comment]
I've tried it with the latest QA build of webextension in Firefox 77.0.1 on mac, on Loebclassics (don't know if the webstie matters or not) and I was not able to reproduce it so far. When I log out, the wordlist tab closes. Tested it logging with both Apple ID and the "regular" (username and password) account, and it worked in both cases.

Did you have any test details different from my test case? I'm wondering what might bring different results.

**Comment by monzug** [comment]
nope, unfortunately. I tried to find a reproducible scenario but not able to.

**Comment by balmas** [comment]
The most likely cause to me so far was a scenario where a large wordlist was still loading when the logout occurred.

**Comment by monzug** [comment]
as Bridget said this is the closest thing to a reproducible scenario:
could it be that I have a huge wordlist that takes some times to load and
that when I am done with the testing and I log out, the wordlist actually
is still loading some section? that would explain the only syriac words in
wordlist...
